.github/CODEOWNERS

```diff
@@ -129,10 +129,10 @@ app/controllers/v0/virtual_agent/virtual_agent_appeal_controller.rb @department-
 app/controllers/v0/virtual_agent/virtual_agent_claim_controller.rb @department-of-veterans-affairs/vfs-virtual-agent-chatbot @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/controllers/v1 @department-of-veterans-affairs/lighthouse-banana-peels
 app/mailers/application_mailer.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/mailers/ch31_submissions_report_mailer.rb @department-of-veterans-affairs/vsa-ebenefits @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+app/mailers/ch31_submissions_report_mailer.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```

Comment:
https://github.com/orgs/department-of-veterans-affairs/teams/benefits-non-disability